### PR TITLE
(6.2) Kubernetes 1.16.0.

### PR DESCRIPTION
**Pull request by r0mant**
Requires https://github.com/gravitational/planet/pull/503. Refs https://github.com/gravitational/gravity.e/issues/4213.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,7 +42,7 @@ RELEASE_OUT ?=
 TELEPORT_TAG = 3.2.7
 # TELEPORT_REPOTAG adapts TELEPORT_TAG to the teleport tagging scheme
 TELEPORT_REPOTAG := v$(TELEPORT_TAG)
-PLANET_TAG := 6.1.5-$(K8S_VER_SUFFIX)
+PLANET_TAG := 6.1.5-$(K8S_VER_SUFFIX)-1-g1b1308e
```
Comment (r0mant) [inline Makefile]: Note to self: don't forget to update planet version.

File: build.assets/robotest_run_nightly.sh [hunk 2]
```diff
@@ -7,7 +7,8 @@ DOCKER_STORAGE_DRIVERS="overlay2"
 
 declare -A UPGRADE_MAP
 # gravity version -> list of OS releases to test upgrades on
-UPGRADE_MAP[5.5.0]="centos:7 ubuntu:16"
+UPGRADE_MAP[6.0.9]="centos:7 ubuntu:16"
```
Comment (a-palchikov) [inline build.assets/robotest_run_nightly.sh]: Is this not working due to version skew policy?

**Comment by a-palchikov** [comment]
Also:
```
update version (6.1.5-1) must be greater than the currently installed version (6.1.5)
```

File: build.assets/robotest_run_nightly.sh (hunk 2 above)
Comment (r0mant) [reply build.assets/robotest_run_nightly.sh]: Yeah. I think it makes sense to test only officially supported upgrade paths.

**Pull request merged**